dbt_project.yml

```diff
@@ -827,9 +827,7 @@ models:
         +schema: ironbank_optimism
 
     keep3r_network:
-      +schema: keep3r_network
-      ethereum:
-        +schema: keep3r_network_ethereum
+      +schema: keep3r_network        
```

Comment:
can you remove lines 829/830 altogether, then add schema property to each model config block next to alias instead?

we're moving towards a cleaner `dbt_project` file in the future, with removing schemas being a big part. easier to keep in the model itself for quick reference to full table name